app/controllers/api_controller/logger.rb

```diff
@@ -1,5 +1,40 @@
 class ApiController
   module Logger
+    def log_request_initiated
+      @requested_at = Time.now.utc
+      api_log_info(" ")
+      log_request("API Request", :requested_at => @requested_at.to_s,
+                                 :method       => request.request_method,
+                                 :url          => request.original_url)
+    end
+
+    def log_api_auth
+      log_request("Authentication", :type        => @auth_token.blank? ? "basic" : "token",
```

Comment:
failed authentications are logged as errors so they will shown up there, i.e. log_api_auth is not called upon failed auth. Thanks.